### PR TITLE
feat(Element): Extend find() to accept > at the start of selector

### PR DESCRIPTION
**Pull request by zofoklecja**
Use element's id to call querySelectorAll - when present, otherwise add temporary one.

### CODE REVIEW

File: src/dom/element.js [hunk 1]
```diff
@@ -490,6 +490,16 @@ class Element {
    */
   find(selector) {
     return this.map(function (node) {
+      if (selector[0] === '>') {
+        const array =  selector.split(' ');
```
Comment (xolir) [inline src/dom/element.js]: Additional space here and i'm not sure array is good name for this variable

File: src/dom/element.js [hunk 2]
```diff
@@ -490,6 +490,16 @@ class Element {
...
+        const el = new Element(node).children(array[1]);
+        if (array.length <= 2) {
+          return el;
+        } else {
+          return el.find(array.splice(2, array.length - 2).join(' '));
```
Comment (xolir) [inline src/dom/element.js]: Could you provide some comments about how this logic proceeds?

File: src/dom/element.js (hunk 1 above)
Comment (zofoklecja) [reply src/dom/element.js]: Yeah, well, that whole algorithm was a mess. Please check the updated version.

File: src/dom/element.js (hunk 2 above)
Comment (zofoklecja) [reply src/dom/element.js]: If the selector starts with > I call children() for this part. If the selector is longer than that, I call find() on it. So for `> foo > bar` I effectively called Element.children(foo).children(bar).
As I said: a Mess.

**Pull request merged**